datamade/la-metro-councilmatic
# Updated readme and docker files

## Overview

Updates `Dockerfile` and `docker-compose.yml` to help run containers locally. Also updates `README.md` with better instructions.

## Testing Instructions

1.  Clone the repo.
2. Follow the README instructions and verify that the containers build/run as expected.

Handles #800
